README.md

```diff
@@ -34,19 +46,30 @@ packages/${CHART_NAME}/
 A regular `package.yaml` will have the following content:
 
 ```yaml
-url: https://charts.bitnami.com/bitnami/external-dns-2.20.10.tgz # url to fetch upstream chart
+url: https://charts.bitnami.com/bitnami/external-dns-2.20.10.tgz # url to fetch upstream chart, remove this key:value for a rancher original chart
 packageVersion: 00 # packageVersion of modified charts, producing a $version-$packageVersion chart. For example, if istio 1.4.7 is modified with changes, rancher produces a 1.4.700 chart version that includes the modification rancher made on top of upstream charts.
+generateCRDChart:
+  enabled: true
```

Comment:
these examples should include the providesGVR field here: https://github.com/rancher/charts/blob/4e030b0ca7ae349ea3b28c4593b00d142da2b80e/packages/rancher-monitoring/package.yaml#L5

Reply:
providesGVR moved outside of `generateCRDChart` and is no longer related to the CRD installation, since CRD installation is handled by a direct chart reference `auto-install: chart=v1.2.3`

Reply:
#604